plugins/parcelport/mpi/CMakeLists.txt

```diff
@@ -53,4 +55,25 @@ if(HPX_WITH_PARCELPORT_MPI)
       ${PROJECT_SOURCE_DIR}
     FOLDER
       "Core/Plugins/Parcelport/MPI")
+elseif(HPX_MPI_WITH_FUTURES)
+  hpx_debug("add_parcelport_mpi_module (reduced)")
+  include(HPX_AddParcelport)
+  add_parcelport(mpi
+    STATIC
+    SOURCES
+      "${PROJECT_SOURCE_DIR}/plugins/parcelport/mpi/mpi_environment.cpp"
+    HEADERS
+      "${PROJECT_SOURCE_DIR}/hpx/plugins/parcelport/mpi/mpi_environment.hpp"
+    DEPENDENCIES
+      hpx_config
+      hpx_command_line_handling
+      hpx_runtime_configuration
+      hpx_synchronization
+      hpx_util
+      hpx_dependencies_boost
+      Mpi::mpi
+    INCLUDE_DIRS
+      ${PROJECT_SOURCE_DIR}
+    FOLDER
+      "Core/Plugins/Parcelport/MPI")
```

Comment:
We should probably move `mpi_environment.cpp` to the MPI module. Having that file here also makes the MPI parcelport not a plugin (even though it's already statically in libhpx).

Reply:
That makes the parcel-port depend on the MPI module, is that what we want?

Reply:
I'd rather have that than the other way around (ok, it's command line handling depending on the MPI parcelport). If necessary we can have `mpi_base` with the MPI environment handling, and have the MPI futures module and the MPI parcelport depend on that.

Reply:
Ok. I'll see what I can do.